_episodes/14-runtime.md

```diff
@@ -39,6 +39,10 @@ To test the above CWL tool use this job to provide the input value `message`:
 ~~~
 {: .source}
 
+Before we run this, lets look at each step in a little more detail.  The base command `baseCommand: ["sh", "example.sh"]` will execute the command `sh example.sh`. This will just run the file we create in the shell.
```

Comment:
I recommend to avoid using the word "just" - it can come across as dismissive language. In this case it can be removed i.e. "This will run the file we create in the shell."